doc/NPCs.md

```diff
@@ -554,6 +554,7 @@ Condition | Type | Description
 `"u_has_any_trait"`<br/>`"npc_has_any_trait"` | array | `true` if the player character or NPC has any trait or mutation in the array. Used to check multiple specific traits.
 `"u_has_var"`, `"npc_has_var"` | string | `"type": type_str`, `"context": context_str`, and `"value": value_str` are required fields in the same dictionary as `"u_has_var"` or `"npc_has_var"`.<br/>`true` is the player character or NPC has a variable set by `"u_add_var"` or `"npc_add_var"` with the string, `type_str`, `context_str`, and `value_str`.
 `"u_compare_var"`, `"npc_compare_var"` | dictionary | `"type": type_str`, `"context": context_str`, `"op": op_str`, `"value": value_num` are required fields, referencing a var as in `"u_add_var"` or `"npc_add_var"`.<br/>`true` if the player character or NPC has a stored variable that is true for the provided operator `op_str` (one of `==`, `!=`, `<`, `>`, `<=`, `>=`) and value.
+`"u_compare_var_time"`, `"npc_compare_var_time"` | dictionary | `"type": type_str`, `"context": context_str`, `"op": op_str`, `"time": time_string` are required fields, referencing a var as in `"u_add_var"` or `"npc_add_var"`.<br/>`true` if the player character or NPC has a stored variable and the current turn and that value (converted to a time point) plus the time_string is true for the provided operator `op_str` (one of `==`, `!=`, `<`, `>`, `<=`, `>=`).  *example*: `{ "u_compare_var_time": "test", "type": "test", "context": "var_time_test", "op": ">", "time": "3 days" }` returns true if the player character has a "test", "test", "var_time_test" variable and the current turn is greater than that value plus 3 days' worth of turns.
```

Comment:
Can we make this just `"compare_var": "test", "type": "time"` so the same syntax can layer be used to compare other numeric variables instead of a whole new almost identical compare_var structure?

Reply:
compare_var and its cousins already lets you do arbitrary additions/subtraction operations on integer values expressed as strings:
```
      {
        "text": "This is a npc_adjust_var test response that decrements by 1.",
        "topic": "TALK_DONE",
        "effect": { "npc_adjust_var": "test", "type": "test", "context": "var_adjust_test", "adjustment": -1 }
      },
      {
        "text": "This is a u_compare_var test response for == 0.",
        "topic": "TALK_DONE",
        "condition": { "u_compare_var": "test", "type": "test", "context": "var_adjust_test", "op": "==", "value": 0 }
      }
```

I'd like it if time comparisons used the same function name as other comparisons, but `"compare_var": { "op": ">", "value": "3" }` checks if a stored value is greater than 3, while `"compare_var_time": { "op": ">", "value": "3 days" }` checks if the current time is greater than the stored value plus 3 days, and the internal logic actually reverse from `compare_var`:

compare_var : stored_value, operator, value (stored value > value)
compare_var_time: time > stored value + value (time > stored value + value)

but I think that's what you want for the most intuitive writing of deadlines.

Reply:
That makes sense then. I didn't realize we already had compare vars. I'll have to get to work on the refugees soon.